Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM silintl/php7:7.2
-MAINTAINER Mark Tompsett <mark_tompsett@sil.org>
+FROM php:7.3-apache-buster
+LABEL maintainer="Mark Tompsett <mark_tompsett@sil.org>"
 
-ENV REFRESHED_AT 2020-05-07
-
-# Fix timezone stuff from hanging.
-RUN apt-get update -y && echo "America/New_York" > /etc/timezone; \
-    apt-get install -y tzdata
+ENV REFRESHED_AT 2021-03-24
 
 # Make sure apt has current list/updates
+RUN apt-get update -y \
+# Fix timezone stuff from hanging.
+    && echo "America/New_York" > /etc/timezone \
```

Comment:
This surprises me. Don't we typically use UTC?

Reply:
Okay, I can change that. Not pushed yet.